extensions/standard-processors/processors/ListenTCP.cpp

```diff
@@ -28,7 +28,13 @@ void ListenTCP::initialize() {
 }
 
 void ListenTCP::onSchedule(core::ProcessContext& context, core::ProcessSessionFactory&) {
-  startTcpServer(context, SSLContextService, ClientAuth);
+  auto delimiter_str = context.getProperty(MessageDelimiter).value_or("\n");
+  if (delimiter_str.empty()) {
+    logger_->log_warn("{} cannot be an empty string", MessageDelimiter.name);
```

Comment:
I think we should tell the user what is happening:
```suggestion
    logger_->log_warn("{} cannot be an empty string, using \\n as the delimiter", MessageDelimiter.name);
```

Reply:
good idea, ive included this in https://github.com/apache/nifi-minifi-cpp/pull/1753/commits/0f488acc7ebffe1ac35e1a0924c96b63a15f5eba